molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/settings/base.py

```diff
@@ -64,11 +64,15 @@
 
     'molo.core',
     '{{cookiecutter.app_name}}',
+    'mptt',
```

Comment:
Tests are failing because of this. You'll need to add `django-mptt` to the `requirements.txt` file.